packages/features/ee/billing/api/webhook/__handler.ts

```diff
@@ -4,29 +4,9 @@ import type { NextApiRequest } from "next";
 import type Stripe from "stripe";
 
 import stripe from "@calcom/features/ee/payments/server/stripe";
-import { HttpError } from "@calcom/lib/http-error";
 
-/** Stripe Webhook Handler Mappings */
-export type SWHMap = {
-  [T in Stripe.DiscriminatedEvent as T["type"]]: {
-    [K in keyof T as Exclude<K, "type">]: T[K];
-  };
-};
-
-export type LazyModule<D> = Promise<{
-  default: (data: D) => unknown | Promise<unknown>;
-}>;
-
-type SWHandlers = {
-  [K in keyof SWHMap]?: () => LazyModule<SWHMap[K]["data"]>;
-};
-
-/** Just a shorthand for HttpError  */
-export class HttpCode extends HttpError {
-  constructor(statusCode: number, message: string) {
-    super({ statusCode, message });
-  }
-}
```

Comment:
Abstracted these to their own files so we're not importing from the main `_handler` file